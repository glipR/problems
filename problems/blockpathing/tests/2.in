1000000 80000000
8000000 30000000
9000000 25000000 13000000 38000000
10000000 7000000 50000000
