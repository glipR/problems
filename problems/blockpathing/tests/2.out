53347224.40622938
